save-common/src/jvmMain/kotlin/org/cqfn/save/entities/BaseEntity.kt

```diff
@@ -0,0 +1,20 @@
+package org.cqfn.save.entities
+
+import javax.persistence.GeneratedValue
+import javax.persistence.GenerationType
+import javax.persistence.Id
+import javax.persistence.MappedSuperclass
+
+/**
+ * base class for all entities
+ */
+@Suppress("USE_DATA_CLASS")
+@MappedSuperclass
+open class BaseEntity {
```

Comment:
Can it be abstract instead of open? I believe it would at least make current error messages (`BaseEntity is not an entity`) more readable